services/bstats/bstats-players.service.js

```diff
@@ -5,7 +5,7 @@ const { metric } = require('../text-formatters')
 const { BaseJsonService } = require('..')
 
 const schema = Joi.array()
-  .items(Joi.array().items([Joi.number().required(), Joi.number().required()]))
+  .items(Joi.array().items(Joi.number().required(), Joi.number().required()))
```

Comment:
Purely for my own edification, what is this? ^^

Reply:
It is looking for is an array of arrays of exactly 2 numbers e.g:

```
[
  [ 1500346800000, 30679 ],
  [ 1567774800000, 110761 ],
  [ 1510439400000, 40127 ]
]
```

We could write it like

```js
const schema = Joi.array()
  .items(
    Joi.array()
      .items(Joi.number().required())
      .length(2)
      .required()
  )
  .required()
```

if you think that is clearer?